cmd/csi-resizer/main.go

```diff
@@ -53,6 +54,8 @@ var (
 	metricsAddress = flag.String("metrics-address", "", "The TCP network address where the prometheus metrics endpoint will listen (example: `:8080`). The default is empty string, which means metrics endpoint is disabled.")
 	metricsPath    = flag.String("metrics-path", "/metrics", "The HTTP path where prometheus metrics will be exposed. Default is `/metrics`.")
 
+	handleVolumeInUseError = flag.Bool("handle-volume-inuse-error", false, "Flag to turn on capability to handle volume in use error in resizer controller")
```

Comment:
We discussed in today's CSI implementation meeting and decided to set it to true by default.

Reply:
Done